src/styles/theme/oc-button.scss

```diff
@@ -26,6 +26,10 @@
     margin-left: $global-margin;
   }
 
+  & {
```

Comment:
Why using `&`? It can be directly in the `.oc-button` styles

Reply:
it didn't work at first, but also I didn't have the `!important`.

trying now...

Reply:
adjusted